pkg/router/httpTriggers.go

```diff
@@ -351,20 +357,26 @@ func (ts *HTTPTriggerSet) updateRouter() {
 		// get triggers
 		latestTriggers := ts.triggerStore.List()
 		triggers := make([]fv1.HTTPTrigger, len(latestTriggers))
+
+		//Iterate over each HTTPTrigger
 		for _, t := range latestTriggers {
 			triggers = append(triggers, *t.(*fv1.HTTPTrigger))
 		}
 		ts.triggers = triggers
 
 		// get functions
 		latestFunctions := ts.funcStore.List()
+		functionTimeout := make(map[types.UID]uint64)
```

Comment:
`make(map[types.UID]uint64)` -> `make(map[types.UID]uint64, len(latestFunctions))` for performance.
btw, please remove the blank line. (L370)

Reply:
Done